StoreApp/Views/Store/Index.cshtml

```diff
@@ -5,18 +5,21 @@
     string role = Session["Role"] as String;
 }
 
-@if (TempData["Message"] != null)
-{
-    <span class="badge badge-success">@TempData["Message"]</span>
-}
-
 @if (role != null && role.Equals("admin"))
 {
     <div class="ol-md-2 offset-2 mb-1">
         @Html.ActionLink("New product", "CreateProduct", "Store", null, new { @class = "btn btn-success" })
     </div>
 }
-<br />
+
+@if (TempData["Message"] != null)
+{
+    <span class="offset-2 badge badge-@TempData["StatusMessage"]">@TempData["Message"]</span>
+}
+else
+{
+    <br />
+}
```

Comment:
Are you sure you need to keep this block? 
Btw you can move tag **br** to the next line after if

Reply:
this stylistic decision